### PR TITLE
Add LTC6268-10 Opamp

### PR DESCRIPTION
**Pull request by cp-aquila**
This adds the single-variants in 6- and 8-pin version.
grafik
grafik

Datasheet: https://www.analog.com/media/en/technical-documentation/data-sheets/626810f.pdf

---

There are quite a lot of travis-violations:
* Text-Size and position (S3.2, S3.6): I choose this to keep the symbol small. Similar devices in the lib have similar violations (`LTC1152`).
* NC pin stack. I choose to make them passive and visible, following the rule for a 'regular' pin-stack. The Datasheet suggest to connect them to a guard ring: 
grafik

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.

### CODE REVIEW

File: Amplifier_Operational.dcm [hunk 1]
```diff
@@ -534,6 +534,18 @@ K quad opamp
...
 $ENDCMP
 #
+$CMP LTC6268xS6-10
+D 4GHz, Ultra-Low Bias Current, FET Input, Op Amp, TSOT-23
+K single opamp transimpedance
+F https://www.analog.com/media/en/technical-documentation/data-sheets/626810f.pdf
+$ENDCMP
+#
+$CMP LTC6268xS8-10
+D 4GHz, Ultra-Low Bias Current, FET Input, Op Amp, SO-8
```
Comment (evanshultz) [inline Amplifier_Operational.dcm]: Change 'SO-8' to 'SOIC-8'.

**Comment by evanshultz** [comment]
Thanks! Otherwise looks fine to me if you can fix the branch conflicts and let the CI tool check for anything I missed.

**Comment by cp-aquila** [comment]
I had to force-push because I rebased onto current master to resolve the merge issue. Sorry about that. The first two commits are unchanged, effectively I just added the 1e4e5f8 commit.

Travis warnings/errors are the same as explained in the first post.

**Comment by evanshultz** [comment]
Looks good  Thanks!

**Pull request merged**